### PR TITLE
Add styling for blockquotes

### LINKED ISSUE #13: Add blockquotes
Blockquote styles are probably the next step! These would probably involve a thick left border of sorts which might need it's own Sass variable.

And let's think about whether or not a footer inside a blockquote should be styled differently for citing a source.

If you decide to take this upon yourself, thanks!

### PR DESCRIPTION
**Pull request by lewinfox**
Fixes #13, adding a $focus sidebar to blockquotes. I haven't looked at styling any nested elements.

### CODE REVIEW

**Comment by kognise** [comment]
We have ourselves a 2.01kb bundle :(

File: index.html [hunk 1]
```diff
@@ -131,6 +131,17 @@ <h4>Heading 4</h4>
...
+    <blockquote cite="hhttps://developer.mozilla.org/en-US/docs/Web/HTML/Element/blockquote">
+        "The HTML blockquote Element (or HTML Block Quotation Element) indicates
+        that the enclosed text is an extended quotation. Usually, this is
+        rendered visually by indentation (see <a
+            href="https://developer.mozilla.org/en-US/docs/Web/HTML/Element/blockquote#Usage_notes">Notes</a>
+        for how to change it). A URL for the source of the quotation may be given using the
+        <code>cite</code> attribute, while a text representation of the source
+        can be given using the <code>&ltcite&gt</code> cite element."
+    </blockquote>
+    <cite>MDN, "The Block Quotation element"</cite>
```
Comment (equinusocio) [inline index.html]: The `<cite>` should be inside the blockquote.

**Comment by kognise** [comment]
Closing in favor of #28 which is based on your styles. Thanks!

**Comment by lewinfox** [comment]
Hi, apologies I haven't had time to improve my initial attempt. Like the changes, looks great! Really like this project, I do a lot of quick prototype pages and will definitely be using this in future :-) Cheers!

**Pull request closed without merging**